docs/support/faq/pouta-openstack-rescue-mode.md

```diff
@@ -2,8 +2,6 @@
 
 In case you run into a situation where your kernel upgrade fails for some reason, and your CentOS 7 instance won't come up, you might find these instructions useful. Sometimes this might seem like a failed resize but the real reason is the restart that a resize requires.
 
-## Symptoms
-
```

Comment:
Why are the subheaders removed?

Reply:
I had some conflict with other changes. I will fix this issues.